numpy/distutils/intelccompiler.py

```diff
@@ -2,7 +2,7 @@
 
 from distutils.unixccompiler import UnixCCompiler
 from numpy.distutils.exec_command import find_executable
-from numpy.distutils.msvc9compiler import MSVCCompiler
+from distutils.msvc9compiler import MSVCCompiler
```

Comment:
Update: on Windows this works because somehow ``msvc9compiler`` ends up in the namespace (likely due to https://github.com/numpy/numpy/blob/master/numpy/distutils/command/config.py#L70), but on Linux the code is actually completely broken.